lib/commands.js

```diff
@@ -3,24 +3,55 @@
 import {CompositeDisposable} from 'event-kit';
 import {packageName, getProjectDirectory} from './util/package-helper';
 
-
 let disposables = null;
-
+let toolBarButtons = {};
 
 export function makeCmdName(name) {
 	return `${packageName()}:${name}`;
 }
 
-export function registerCommands(atom,
+export function registerCommands(atom, toolBar,
```

Comment:
It feels like we are mixing things up here..I'd probably have configured the toolbar externally, or moved it to a separate method.  But it's not a significant issue.